nixos/tests/all-tests.nix

```diff
@@ -412,7 +412,9 @@ in {
   mpv = handleTest ./mpv.nix {};
   mtp = handleTest ./mtp.nix {};
   mumble = handleTest ./mumble.nix {};
-  musescore = handleTest ./musescore.nix {};
+  # Fails on aarch64-linux at the PDF creation step - need to debug this on an
+  # aarch64 machine..
+  musescore = handleTestOn ["x86_64-linux"] ./musescore.nix {};
```

Comment:
But we can test on darwin as well, right?

Reply:
> But we can test on darwin as well, right?

I don't think so - in it's current form, it uses xserver to launch musescore. I'm not sure at all if there are nixos tests that are meant for Darwin systems..

Reply:
If I understand correctly, according to this file:

https://github.com/NixOS/nixpkgs/blob/31968e86eddf260716458ee9ede65691f6e1987f/nixos/release.nix#L5-L27

only `{x86_64,aarch64}-linux` tests are performed on Hydra, meaning that the current exclusion with `handleTestOn` should be OK.